tools/nni_cmd/nnictl_utils.py

```diff
@@ -681,45 +682,67 @@ def monitor_experiment(args):
     set_monitor(False, args.time)
 
 def export_trials_data(args):
-    '''export experiment metadata to csv
+    '''export experiment metadata and intermediate results to json or csv
     '''
+    def groupby_trial_id(intermediate_results):
+        sorted(intermediate_results, key=lambda x: x['timestamp'])
+        groupby = dict()
+        for content in intermediate_results:
+            groupby.setdefault(content['trialJobId'], []).append(content['data'][2:-2])
+        return groupby
+
+    def trans_intermediate_dict(record):
+        return {'intermediate': '[' + str(reduce(lambda x, y: x + ',' + y, record)) + ']'}
```

Comment:
i don't know the content of `record`, but could we use `','.join(record)` instead of `str(reduce(lambda x, y: x + ',' + y, record))`?

Reply:
what is the content of record?

Reply:
`record` is a list with float numbers in it.
`','.join()` is a good solution.

Reply:
I have not tested intermediate results with dict metric. I will do it this afternoon.